README.md

```diff
@@ -167,6 +167,9 @@ Mona uses several environment variables you can set as you prefer:
 - MONA_SDK_SHOULD_USE_AUTHENTICATION - When set to false, the communication with Mona's servers will not use 
   authentication (user_id should be provided to the Client constructor instead of an api_key and a secret), **Note**: 
   this mode is not supported on the servers by default, and must be explicitly requested from Mona's team.
+- MONA_SDK_FILTER_NONE_FIELDS_ON_EXPORT - When set to true, Mona's client will filter out all fields with None values 
+  from the message dict to export. There is an addition argument on both export() and export_batch() called 
```

Comment:
"There is an addition argument on both export() and export_batch() called filter_none_fields that ca be used to override this value for a single exported message/batch" ->
"Note that passing a None value may be required in order to delete a pre-existing value. To allow None values, use filter_none_fields=False which overrides this parameter both in export() and export_batch() functions.